DaeguGoodPriceShop/DaeguGoodPriceShop/ViewModel/MapViewModel.swift

```diff
@@ -81,7 +81,7 @@ final class MapViewModel {
     }
     
     func getShops() -> [Shop] {
```

Comment:
```suggestion
    func totalShops() -> [Shop] {
```

p4;

Swift 는 반환하는 값을 명확하게 표현할 수 있는 경우 get~ 네이밍을 지양하고 있습니다.
이와 같이 네이밍을 변경하거나 연산 프로퍼티를 사용하는 것을 고려해보셨을까요?

Reply:
연산 프로퍼티로 totalShops로 하면 깔끔하겠네요.